### DISCUSSION TITLE
Making armor appear when in curios slot

### DISCUSSION THREAD

**Discussion by xIronxS** [TheIllusiveC4/Curios#316] [Q&A]
I am trying to make a datapack for curios that lets me put certain armor in the curios slots. I have gotten that part to work, however, when they are in the curios slot, they do not show up on the player model or function in any way. If there is a way I can fix this or do that, I'd love to know. Thank you

**Comment by TheIllusiveC4** [comment]
Unfortunately, something complex like that would require coding a mod in order to properly render the model and determine how it functions while in the slot(s).

**Reply by xIronxS** [reply]
Dang, thanks anyway then!

**Comment by amonNihrodskiy** [comment]
Then what is the meaning of the datapack? If the item placed in the slot using the datapack does not work in any way, is not displayed and does not do anything?

**Reply by TheIllusiveC4** [reply]
Items that do not have a curio function can still be stored in those slots, so it has value as extra storage.

More to the point, the purpose of the datapack features is so that users and developers can change and manipulate items to go into different slots. For instance, a ring item that already has a curio function can be changed to go into the necklace slot instead and it will function from there as it would from the ring slot. In all cases, the function provided needs to come from a mod in order to define the curio behavior being used.
